PR-Issue#153 "Add method to upload points to the API"

gabrielalexandro opened the pull request.

tcrurav reviewed (approved):
Hi Gabriel,

You have deleted the file backend/.env.example which shouldn't be deleted.

But anyway I approve this PR. I will add again this file in a next PR.

Thank you.

gabrielalexandro commented:
I'm really sorry, I didn't notice.

I had to clone the entire project again, and instead of duplicating the
".env.example," I just renamed it and added my credentials.

I will be more careful next time. Thank you.

El vie., 14 feb. 2025 10:10 a. m., Tiburcio Cruz ***@***.***>
escribió:

> ***@***.**** approved this pull request.
>
> Hi Gabriel,
>
> You have deleted the file backend/.env.example which shouldn't be deleted.
>
> But anyway I approve this PR. I will add again this file in a next PR.
>
> Thank you.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/tcrurav/xreate/pull/154#pullrequestreview-2617152583>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/BLXSY626TJ6HAROL33WBCOL2PWXJTAVCNFSM6AAAAABXD34XQ2VHI2DSMVQWIX3LMV43YUDVNRWFEZLROVSXG5CSMV3GSZLXHMZDMMJXGE2TENJYGM>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>
